frontend/src/metabase/query_builder/components/view/NativeCodePanel/NativeCodePanel.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { t } from "ttag";
+import Icon from "metabase/components/Icon";
+import Tooltip from "metabase/components/Tooltip";
+import {
+  CodeContainer,
+  CodeCopyButton,
+  CodeRoot,
+  CodeText,
+} from "./NativeCodePanel.styled";
+
+interface NativeCodePanelProps {
+  value: string;
+  isHighlighted?: boolean;
+  isCopyEnabled?: boolean;
+}
+
+const NativeCodePanel = ({
+  value,
+  isHighlighted,
+  isCopyEnabled,
+}: NativeCodePanelProps): JSX.Element => {
+  const { isCopied, handleCopy } = useCopyButton(value);
+
+  return (
+    <CodeRoot>
+      <CodeContainer isHighlighted={isHighlighted}>
+        <CodeText>{value}</CodeText>
+      </CodeContainer>
+      {isCopyEnabled && (
+        <Tooltip tooltip={t`Copied!`} isOpen={isCopied}>
+          <CodeCopyButton isHighlighted={isHighlighted} onClick={handleCopy}>
+            <Icon name="copy" />
```

Comment:
Not sure if this is the desired icon.

#### PR
<img width="131" alt="image" src="https://user-images.githubusercontent.com/1937582/206561586-8bc22d7d-5e9c-4dce-9b89-833ce3392639.png">

#### Design 
<img width="110" alt="image" src="https://user-images.githubusercontent.com/1937582/206561620-ac46d2f9-f0f7-4800-98ef-d85c2fb91758.png">